node/client/src/lib.rs

```diff
@@ -500,8 +496,7 @@ where
 		+ sp_api::Metadata<Block>
 		+ sp_offchain::OffchainWorkerApi<Block>
 		+ sp_session::SessionKeys<Block>
-		+ sp_authority_discovery::AuthorityDiscoveryApi<Block>,
-	<Self as sp_api::ApiExt<Block>>::StateBackend: sp_api::StateBackend<BlakeTwo256>,
```

Comment:
This change is introduced by substrate [PR #14612](https://github.com/paritytech/substrate/pull/14612)